src/shared-components/alert/index.tsx

```diff
@@ -28,18 +28,14 @@ const alertIconMapping = {
 export type AlertType = 'success' | 'error' | 'default' | 'danger';
 
 type AlertProps = {
-  avatarSrc: string;
+  avatarSrc?: string;
   content: React.ReactNode;
-  ctaContent: React.ReactNode;
-  duration: string | number;
-  truncateText: boolean;
-  type: AlertType;
-  onExit: (rest: Omit<AlertProps, 'onExit'>) => void | (() => void);
-};
-
-type AlertState = {
-  exiting: boolean;
-  exited: boolean;
+  ctaContent?: React.ReactNode;
+  duration?: string | number;
+  truncateText?: boolean;
+  type?: AlertType;
+  onExit?: () => void;
```

Comment:
[nit] Alphabetically, this should come after `duration` 🙂

Reply:
I swear I passed over the files before pushing like 5 times and I was sure I would have missed some